core/internal/cltest/cltest.go

```diff
@@ -1648,3 +1648,17 @@ func MustGetStateForKey(t testing.TB, kst keystore.Eth, key ethkey.KeyV2) ethkey
 func NewBulletproofTxManagerORM(t *testing.T, db *sqlx.DB, cfg pg.LogConfig) bulletprooftxmanager.ORM {
 	return bulletprooftxmanager.NewORM(db, logger.TestLogger(t), cfg)
 }
+
+// ClearDBTables deletes all rows from the given tables
+func ClearDBTables(t *testing.T, db *sqlx.DB, tables []string) {
```

Comment:
Maybe a good case for variadic? Then the callers can be a bit simpler.
```suggestion
func ClearDBTables(t *testing.T, db *sqlx.DB, tables ...string) {
```

Reply:
Nice change!